handlers/validate/validators/collection/collection_test.go

```diff
@@ -0,0 +1,53 @@
+package collection
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/hashicorp/go-version"
+)
+
+func Test_isUnsupportedVersion(t *testing.T) {
+	v, err := version.NewVersion("1.2.3")
+	require.NoError(t, err)
+
+	tests := []struct {
+		name                string
+		unsupportedVersions []string
+		want                bool
+	}{
+		{
```

Comment:
I suggest to differentiate 3 test cases:
- nil unsupported versions (edge case check)
- 2 unsupported version + version included 
- 2 unsupported version + version NOT included